labrad/util/__init__.py

```diff
@@ -370,7 +375,15 @@ def runServer(srv, run_reactor=True, stop_reactor=True):
 
     @inlineCallbacks
     def run(srv):
-        reactor.connectTCP(config['host'], int(config['port']), srv)
+        host = config['host']
+        port = int(config['port'])
+        if config['use-tls']:
+            tls_options = crypto.tls_options(host)
+            reactor.connectSSL(host, port, srv, tls_options)
+        else:
+            srv._start_tls = config['tls'].lower() != 'off'
```

Comment:
It's odd to see _variables referenced outside of the class itself. If I refactored the server class I'd assume that all _variables can be changed without having to worry about updating other files or tests.

I'd make this a non-internal variable or even better a function: srv.use_tls(true) or srv.use_stl(false) or have the server update itself from the config dictionary.